### PR TITLE
Add build instructions to Readme

### PR DESCRIPTION
**Pull request by 505e06b2**
A bit of a low-hanging fruit, but I think people would find this helpful

If there's anything needing changed before the merge, I'd be happy to sort it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
 # UnionPatcher
 
-A tool for LittleBigPlanet that patches the server urls to a custom url.
+A commandline tool that will take the official EBOOT of a LittleBIGPlanet title and replace the server URLs with the ones specified
```
Comment (jvyden) [inline README.md:3]: While UnionPatcher does have a command line utility, there *is* also a GUI.

Also, this sentence reads a bit worse in general.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
 
-# Downloading
-You can download UnionPatcher by going [here](https://github.com/LBPUnion/UnionPatcher/actions), clicking the top link, and choosing a download for your platform at the bottom.
-Mac will have to be self-compiled for now, but it does work.
+## Prerequisites
+* For running the console application, you will need the [.NET 6 Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/6.0/runtime?utm_source=getdotnetcore&utm_medium=referral)
+* You will need to know the server URL you wish to use
+* Basic commandline knowledge
```
Comment (jvyden) [inline README.md:8]:
```suggestion
```

**Review by jvyden — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (505e06b2) [reply README.md]: Removed the references to the commandline

File: README.md (hunk 2 above)
Comment (505e06b2) [reply README.md]: Removed the references to the commandline

**Review by jvyden — APPROVED** [review]

**Pull request merged**